R/theme.R

```diff
@@ -2,10 +2,26 @@ build_bslib <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
   bs_theme <- bs_theme(pkg)
 
+  cur_deps <- find_deps(pkg)
+  cur_digest <- purrr::map_chr(cur_deps, file_digest)
+
   deps <- bslib::bs_theme_dependencies(bs_theme)
   deps <- lapply(deps, htmltools::copyDependencyToDir, file.path(pkg$dst_path, "deps"))
   deps <- lapply(deps, htmltools::makeDependencyRelative, pkg$dst_path)
 
+  new_deps <- find_deps(pkg)
+  new_digest <- purrr::map_chr(cur_deps, file_digest)
+
+  all_deps <- union(new_deps, cur_deps)
+  diff <- cur_digest[all_deps] == new_digest[all_deps]
```

Comment:
```suggestion
  diff <- (cur_digest[all_deps] == new_digest[all_deps])
```

this would be my personal preference